docs/threat_models/v0.17.4-beta.json

```diff
@@ -0,0 +1,751 @@
+{
+  "version": "2.2.0",
+  "summary": {
+    "title": "Spice.ai OSS",
+    "owner": "Phillip LeBlanc",
+    "description": "Spice is a portable, open-source runtime for fast, last-mile SQL query and AI inference.",
+    "id": 0
+  },
+  "detail": {
+    "contributors": [],
+    "diagrams": [
+      {
+        "id": 0,
+        "title": "SQL Query (Sidecar)",
```

Comment:
This shouldn't be sidecar specific. That's a made up term anyway. It should just be virtual/physical machine/container boundary.